plugin/storage/integration/grpc_test.go

```diff
@@ -164,6 +164,31 @@ func TestGRPCStorage(t *testing.T) {
 	s.IntegrationTestAll(t)
 }
 
+func TestGRPCStreamingWriter(t *testing.T) {
+	binaryPath := os.Getenv("PLUGIN_BINARY_PATH")
+	if binaryPath == "" {
+		t.Logf("PLUGIN_BINARY_PATH env var not set, using %s", defaultPluginBinaryPath)
+		binaryPath = defaultPluginBinaryPath
+	}
+	configPath := os.Getenv("PLUGIN_CONFIG_PATH")
+	if configPath == "" {
+		t.Log("PLUGIN_CONFIG_PATH env var not set")
+	}
+
+	flags := []string{
+		"--grpc-storage-plugin.binary", binaryPath,
+		"--grpc-storage-plugin.log-level", "debug",
+		"--grpc-storage-plugin.writer-type", "streaming",
+		"--grpc-storage-plugin.configuration-file", configPath,
+	}
```

Comment:
this is largely repeating the code from the previous function; let's move into a helper `getPluginFlags`

Reply:
also, the actual values seem identical to the previous test, did you want to include the new option here? It can be appended after calling getPluginFlags()

Reply:
done